plugins/depthcamera/DepthCameraDriver.cpp

```diff
@@ -0,0 +1,273 @@
+#include "DepthCameraDriver.h"
+
+
+using namespace yarp::dev::gzyarp;
+
+
+YARP_LOG_COMPONENT(GZDEPTH, "gz-sim-yarp-plugins.plugins.GzYarpDepthCamera")
+
+
+bool DepthCameraDriver::open(yarp::os::Searchable& config)
+{
+    yCDebug(GZDEPTH) << "Opening Gazebo Yarp Depth Camera Driver";
```

Comment:
gz-sim-yarp-plugins is already quite verbose, perhaps we can avoid to leave around this debug messagges?